### PR TITLE
Qualify all field/method access

### PR DESCRIPTION
**Pull request by hazendaz**
As part of an effort to apply checks that eclipse can flag, qualify all access which additionally makes it easier to understand by look at code where object in use comes from.

### CODE REVIEW

**Comment by hazendaz** [comment]
@jander99 @MALfunction84 

If  possible can I get an ok on this so I can then bring in a major fix around issue #687 which is the long standing issue I have been mentioning where grids didn't look right.  Found the issue.  I indirectly introduced it back in July of last year.  I'll be making a pull request on 2.4.x branch as well to fix the problem and plan to release 2.4.1.

**Comment by MALfunction84** [comment]
I generally don't like large changes, and I think this specific change is not necessary. Usage of `this` that is not required makes the code busier and harder to read.

If a local variable hides a field, you should just rename one of them. Many IDEs provide a warning for this. (I actually have my syntax highlighting set up to color fields and local variables differently.)

My one exception to this rule is for field assignments:

    class Example {
      Object value;

      Example(Object value) {
        this.value = value; // OK
      }   

      void setValue(Object value) {
        this.value = value; // OK
      }

      void doSomething() {
        Object value = new Object(); // not OK, hides field
      }
    }

In short, I don't like this change. Can you create a separate PR for fixing #687?
